cvat/apps/engine/views.py

```diff
@@ -616,10 +614,38 @@ def data(self, request, pk):
                     status=status.HTTP_400_BAD_REQUEST)
             serializer = DataSerializer(data=request.data)
             serializer.is_valid(raise_exception=True)
+            data = {k: v for k, v in serializer.validated_data.items()}
+
+            tmp_path = os.path.join(settings.TMP_DATA_ROOT, pk)
+            os.makedirs(tmp_path, exist_ok=True)
```

Comment:
@klakhov , What if several tasks are created in parallel? Can they override data of each other?

Reply:
@nmanovic, There are no problems with that. There is separate tmp directory for each task in `data/tmp/<tid>`